packages/dd-trace/src/appsec/blocking.js

```diff
@@ -150,5 +154,6 @@ module.exports = {
   specificBlockingTypes,
   getBlockingData,
   getBlockingAction,
-  setTemplates
+  setTemplates,
+  responseBlockedSet
```

Comment:
instead of exporting responseBlockedSet this module could export an isBlocked(res) function

Reply:
even being a WeakSet, should we delete the response entry from the set once request is finished?

Reply:
No, we can't do it. It is just to prevent apps sending response when we end the response. We can't remove response from the weakset when we finish the response, it will loose its purpose

Reply:
okis